ember-toucan-core/src/components/form/controls/textarea.hbs

```diff
@@ -1,7 +1,11 @@
 <textarea
-  class="min-h-6 bg-overlay-1 focus:outline-none focus:shadow-focus-outline block h-20 rounded-sm p-1
+  class="min-h-6 bg-overlay-1 focus:outline-none focus:shadow-focus-outline block h-20 rounded-sm p-1 transition-shadow
     {{if @isDisabled 'text-disabled' 'text-titles-and-attributes'}}
-    {{if @hasError 'shadow-error-outline' 'shadow-focusable-outline'}}"
+    {{if
+      @hasError
+      'shadow-error-outline focus:shadow-error-focus-outline'
```

Comment:
Should we update the related [tests](https://github.com/CrowdStrike/ember-toucan-core/blob/main/test-app/tests/integration/components/textarea-test.gts#L93) to test for `focus:shadow-error-focus-outline`?

Reply:
but actually wait...how would you test for this? The computed property?

Reply:
Excellent callout - went ahead and added some tests!  We can assert the class is there, but can't verify it looks how we expect.  For that, we will need Percy or something similar to do visual regression testing! Soon™?!